ui/src/clientLibraries/graphics/GoogleLogo.tsx

```diff
@@ -0,0 +1,27 @@
+// Libraries
+import React, {FC} from 'react'
+
+interface Props {
+  className?: string
+}
+
+export const GoogleLogo: FC<Props> = ({className}) => (
+  <svg className={`google-logo ${className}`} x="0" y="0" viewBox="0 0 17.6 18">
+    <path
+      d="M15 15.8h-3v-2.3c1-.6 1.6-1.6 1.8-2.7H9V7.4h8.5c.1.6.2 1.2.2 1.8-.1 2.7-1 5.1-2.7 6.6z"
+      className="google-blue"
```

Comment:
not sure, but @alexpaxton might have some opinions on using BEM style naming here (class should represent the structure, so like: `google-logo--blue-thing`)